web/themes/contrib/civictheme/templates/paragraphs/paragraph--civictheme-manual-list.html.twig

```diff
@@ -9,13 +9,15 @@
 {% block rows %}
   {% if rows -%}
     <div class="ct-list__rows">
-      {% include '@base/item-grid/item-grid.twig' with {
+      {% include '@base/grid/grid.twig' with {
```

Comment:
for cases where we need to generate markup, we are still using a twig file rather than just CSS classes.